packages/mgt-components/src/components/mgt-people/mgt-people.ts

```diff
@@ -415,6 +436,30 @@ export class MgtPeople extends MgtTemplatedComponent {
           this.people = await getPeople(graph);
         }
 
+        //replace people with fallback details
+        if (this._fallbackDetails) {
```

Comment:
I think we can simplify this if we make an assumption that the fallbackDetails provided by the developer are in the same order as the userIds. Instead of searching for an id which the fallbackDetails might not have, we can simply look for null people in `this.people` and replace those with the `fallbackDetails[i]` value.

```ts
if (this._fallbackDetails) {
  // replace null people with fallback details
  this.people = this.people.map((p, i) => {
    if (p) {
      return p;
    } else if (i < this._fallbackDetails.length) {
      return this._fallbackDetails[i];
    }
    return null;
  });
}
```

And this should only be done if `this.people` is created from userIds, so move up a bit inside that code block.

Reply:
With those requirements, that makes sense. I'll update the documentation as well, with the "displayname" for initials disclaimer. 

updated here:

5e8f43d